newrelic/hooks/external_aiobotocore.py

```diff
@@ -11,9 +11,34 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import logging
+import traceback
+import sys
+from aiobotocore.response import StreamingBody
+from io import BytesIO
 
 from newrelic.api.external_trace import ExternalTrace
 from newrelic.common.object_wrapper import wrap_function_wrapper
+from newrelic.hooks.external_botocore import (
+    handle_bedrock_exception,
+    run_bedrock_response_extractor,
+    run_bedrock_request_extractor,
+    RESPONSE_PROCESSING_FAILURE_LOG_MESSAGE,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+# Class from https://github.com/aio-libs/aiobotocore/blob/master/tests/test_response.py
+class AsyncBytesIO(BytesIO):
```

Comment:
Can you link the original license for that repo as well? Pretty sure that's a requirement for Apache 2.